src/dbmanager/dbmanager.go

```diff
@@ -276,3 +276,46 @@ func (dbmanager *DBManager) AddRoute(r *domain.Route) (int, error) {
 	}
 	return int(idRoute), nil
 }
+
+//TakePlace takes one place in bus
+func (dbmanager *DBManager) TakePlace(id int) (*domain.Ticket, error) {
+	row, err := dbmanager.db.Query(`SELECT r.id_route, r.starttime, r.cost, r.freeseats, r.allseats, 
+	p.id_points, p.startpoint, p.endpoint 
+	FROM route r JOIN points p on r.id_points = p.id_points WHERE r.id_route=?`, id)
+	if err != nil {
+		return nil, errors.New("data hasn't read")
+	}
+
+	if !row.Next() {
+		return nil, errors.New("no such route")
```

Comment:
You can make const error here too. And move this error to domain package, so you can use it in different packages.